lib/assertions.ts

```diff
@@ -1,14 +1,16 @@
 import exists from './assertions/exists';
 import focused from './assertions/focused';
-import notFocused from './assertions/not-focused';
 import isChecked from './assertions/is-checked';
+import isDisabled from './assertions/is-disabled';
 import isNotChecked from './assertions/is-not-checked';
-import isRequired from './assertions/is-required';
+import isNotOverflowing from './assertions/is-not-overflowing';
 import isNotRequired from './assertions/is-not-required';
+import isOverflowing from './assertions/is-overflowing';
+import isRequired from './assertions/is-required';
```

Comment:
I agree that ordering these alphabetically probably makes sense, but that is also unrelated to adding the feature here. if you want to refactor things that are unrelated to the feature then I would recommend doing so in dedicated PRs (or at least commits), so that these changes are easier to review.

for more info on these topics have a look at https://mtlynch.io/code-review-love/ :)

Reply:
Thanks for sending this over! I'll undo the sorting here.